src/lib/db.ts

```
@@ -0,0 +1,56 @@
+	// Enable debug mode in development
+	...(process.env.NODE_ENV === 'development' && {
+		debug: (connection, query, params) => {
+			console.log("Running SQL Query:", query);
+			console.log("📌 Query Parameters:", params);
+		},
+	}),
+});
+
+// Connect Drizzle ORM
+export const db = drizzle(client);
+export const db = drizzle(client);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'db'. Consider to delete it or rename it.

'db' is defined here:


(lint/suspicious/noRedeclare)